mevolib/cluster/PRD.py

```diff
@@ -77,7 +76,7 @@ def _prd_decomposition ( tree_file, subset_size, overlapping, binary = 'dcm' ) :
     # Build the command line and launch it handling any Runtime exception
     command = [binary, '3', tree_file, overlapping]
     try :
-        output = subprocess.check_output(command, stderr=DEVNULL,
+        output = subprocess.check_output(command, stderr=os.devnull,
```

Comment:
It is best to use `subprocess.DEVNULL` to make the code more future-proof:
```suggestion
        output = subprocess.check_output(command, stderr=subprocess.DEVNULL,
```